db/migrations/000002_create-events-table.up.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE IF NOT EXISTS organizations (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(100) NOT NULL DEFAULT 'self'
+);
+
+CREATE TABLE IF NOT EXISTS images (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    filename VARCHAR(255),
+    url VARCHAR(255),
+    alt VARCHAR(150),
+    uploaded_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS locations (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL DEFAULT 'TBD',
+    url VARCHAR(255)
+);
+
+CREATE TABLE IF NOT EXISTS event_more_info (
```

Comment:
I would lean towards keeping these fields as part of the events table. Maybe as `additional_info` and `additional_url`. Is there a reason for having this in a separate table?

Reply:
I'm not sure what led me to create this additional table. This table is unnecessary, its only purpose was to minimize the amount of columns in `events`.